### PR TITLE
Adding ASP.NET 5 topic links - Issue #860 - erikre

### PR DESCRIPTION
**Pull request by Erikre**

### CODE REVIEW

**Comment by Erikre** [comment]
I made changes based on the following details:

Title: Application Insights for ASP.NET 5
URL: http://azure.microsoft.com/documentation/articles/app-insights-asp-net-five/ 
Action: Added as additional resource link to the following topic: Application Insights

Title: Create an ASP.NET 5 web app in Visual Studio Code 
URL: http://azure.microsoft.com/documentation/articles/web-sites-create-web-app-using-vscode/ 
Action: Added to Tutorials section.

Title: Get started with Azure Blob storage and Visual Studio connected services (ASP.NET 5) 
URL: http://azure.microsoft.com/documentation/articles/vs-storage-aspnet5-getting-started-blobs/ 
Action: Added to the Working with Data section.

Title: Get started with queue storage and Visual Studio connected services (ASP.NET 5) 
URL: http://azure.microsoft.com/documentation/articles/vs-storage-aspnet5-getting-started-queues/
Action: Added to the Working with Data section.

Title: How to get started with Azure Table storage and Visual Studio connected services 
URL: http://azure.microsoft.com/documentation/articles/vs-storage-aspnet5-getting-started-tables/
Action: Added to the Working with Data section.

Title: Adding Azure storage by using Visual Studio Connected Services 
URL: http://azure.microsoft.com/documentation/articles/vs-azure-tools-connected-services-storage/
Action: Added to the Working with Data section.

Title: Build a web service front-end for your application 
URL: http://azure.microsoft.com/documentation/articles/service-fabric-add-a-web-frontend/ 
Action: Added to the Tutorials section.

Title: Hosting Web Apps in Docker 
URL: http://azure.microsoft.com/documentation/articles/vs-azure-tools-docker-hosting-web-apps-in-docker/ 
Action: Added to the Publishing and Deployment section.

**Comment by Erikre** [comment]
Please review - @danroth27 @rachelappel @Rick-Anderson

File: aspnet/fundamentals/application-insights.rst [hunk 1]
```diff
@@ -121,4 +121,5 @@ Additional Resources
 --------------------
 
 - `Application Insights API for custom events and metrics <http://azure.microsoft.com/documentation/articles/app-insights-api-custom-events-metrics/>`_
-- `Application Insights for ASP.NET 5 <http://blogs.msdn.com/b/webdev/archive/2015/05/19/application-insights-for-asp-net-5-you-re-in-control.aspx>`_
+- `Application Insights for ASP.NET 5 <https://azure.microsoft.com/en-us/documentation/articles/app-insights-asp-net-five/>`_
+- `Application Insights - Blog post <http://blogs.msdn.com/b/webdev/archive/2015/05/19/application-insights-for-asp-net-5-you-re-in-control.aspx>`_
```
Comment (danroth27) [inline aspnet/fundamentals/application-insights.rst]: Remove - no links to blog posts allowed. Blog posts are point-in-time content. We want our docs to be more stable than that.

File: aspnet/publishing/index.rst [hunk 2]
```diff
@@ -13,3 +13,4 @@ Publishing and Deployment
     how-to-customize
     iis-with-msdeploy
     web-publishing-vs
+	Hosting Web Apps in Docker <https://azure.microsoft.com/documentation/articles/vs-azure-tools-docker-hosting-web-apps-in-docker/>
```
Comment (danroth27) [inline aspnet/publishing/index.rst]: This docker doc appears to be a close duplicate of https://docs.asp.net/en/latest/publishing/docker.html. I don't think we need both. Which one should we keep?

File: aspnet/tutorials/index.rst [hunk 3]
```diff
@@ -10,4 +10,5 @@ Tutorials
    publish-to-azure-webapp-using-vs
    nano-server
    ASP.NET 5 and Azure Service Fabric <https://azure.microsoft.com/en-us/documentation/articles/service-fabric-add-a-web-frontend/>
-   
+   Create an ASP.NET 5 web app in Visual Studio Code <https://azure.microsoft.com/documentation/articles/web-sites-create-web-app-using-vscode/>
+   Build a web service front-end for your application <https://azure.microsoft.com/documentation/articles/service-fabric-add-a-web-frontend/>
```
Comment (danroth27) [inline aspnet/tutorials/index.rst]: Isn't this a duplicate of the one of the links already in the TOC? See line #12.

**Comment by danroth27** [comment]
@Erikre Please redo this PR as a server side branch so that we can continue to iterate on it as needed.

**Pull request closed without merging**